### DISCUSSION TITLE
Sorce linking of documents in local search.

### DISCUSSION THREAD

**Discussion by evozondatascience** [microsoft/graphrag#1433] [Q&A]
Hello everyone, I am trying to play with local search and make the llm to tell me at the end of a response the title of the document from where it took all the data. I saw that the build_context function will return some context_data where you can find all Entities, Comunities and Text_units used. 
I was wondering if there is any function that could help me to also get the documents or should I implement one. Having the text_units_ids and create_final_documents file would be enough. 
I just wanted to make sure if there is a flow already implemented for this.

**Comment by godwin05** [comment]
i think , that is possible , if u change the some prompts in QA and add same additional column in parquet files
